openhabian-setup.sh

```diff
@@ -61,7 +62,6 @@ if [[ -n "$UNATTENDED" ]]; then
   timezone_setting
   locale_setting
   hostname_change
-  if is_pi; then memory_split; enable_rpi_audio; fi
```

Comment:
Would leave memory_split up here. No real reason, just for the sake of not changing more than meaningful

Reply:
That would mean to clutter functionality.
As memory_split() only writes into /boot/config.txt it does not really matter when it does that.